### PR TITLE
adding check-manifest hook

### PR DESCRIPTION
**Pull request by drahc1R**
adding check-manifest hook

### CODE REVIEW

File: MANIFEST.in [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+examples/marginal_dist_detection.ipynb
+examples/noise/check_noise_correlation.py
+examples/replicate_null_demo.ipynb
+examples/sample_datasets/01_generate_datasets.py
+examples/sample_datasets/diamonds.csv
+examples/synth_graph_data/demo_graph.csv
+examples/synth_graph_data/graph_data_demo.ipynb
+generators_UML_diagram.pdf
+requirements-test.txt
+requirements.txt
```
Comment (taylorfturner) [inline MANIFEST.in:47]: new line EOF

File: setup.cfg [hunk 2]
```diff
@@ -10,3 +10,21 @@ line_length=88
...
+
+[check-manifest]
+ignore-default-rules=True
+ignore =
+    synthetic_data.egg-info/**
+    synthetic_data/__pycache__/**
+    docs
+    PKG-INFO
+    setup
+    setup/**
```
Comment (taylorfturner) [inline setup.cfg:22]: do we need `setup`?

File: setup.cfg [hunk 3]
```diff
@@ -10,3 +10,21 @@ line_length=88
...
+    tests/**
+    .pytest_cache/**
+    .pre-commit-config.yaml
```
Comment (taylorfturner) [inline setup.cfg:25]: probably don't need the `setup.cfg` too

File: setup.cfg (hunk 2 above)
Comment (taylorfturner) [reply setup.cfg:22]: do we need this though?

Comment (drahc1R) [reply setup.cfg:22]: hmm, should i remove this?

**Review by taylorfturner — APPROVED** [review]

**Review by JGSweets — APPROVED** [review]

**Pull request merged**